### PR TITLE
chore: Test sharing to cloud

### PR DESCRIPTION
**Pull request by sklein12**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -260,3 +260,8 @@ jobs:
             echo "Error: Expected 1 entry, but got $count"
             exit 1
           fi
+
+      - name: Share to cloud
+        run: |
+          node dist/src/main.js auth login -k ${{ secrets.PROMPTFOO_STAGING_API_KEY }} -h https://api.promptfoo-staging.app
```
Comment (mldangelo) [inline .github/workflows/main.yml:266]: maybe we should `npm link` promptfoo although that may be more ambiguous than doing it this way.

File: src/globalConfig/cloud.ts [hunk 2]
```diff
@@ -69,7 +69,10 @@ export class CloudConfig {
     };
   }
 
-  async validateAndSetApiToken(token: string, apiHost: string): Promise<void> {
+  async validateAndSetApiToken(
+    token: string,
+    apiHost: string,
+  ): Promise<{ user: any; organization: any; app: any }> {
```
Comment (mldangelo) [inline src/globalConfig/cloud.ts]: can we type this better?

**Review by mldangelo — APPROVED** [review]
Thanks @sklein12!

File: .github/workflows/main.yml (hunk 1 above)
Comment (sklein12) [reply .github/workflows/main.yml:266]: yea, i just wanted to ensure we were running the right files

File: src/globalConfig/cloud.ts (hunk 2 above)
Comment (sklein12) [reply src/globalConfig/cloud.ts]: done!

**Pull request merged**